build.sbt

```diff
@@ -276,6 +277,24 @@ lazy val tensorflow: Project = project
     test % "test->test"
   )
 
+lazy val protobuf: Project = project
+  .in(file("protobuf"))
+  .settings(
+    commonSettings,
+    moduleName := "magnolify-protobuf",
+    description := "Magnolia add-on for Apache Avro",
+    libraryDependencies ++= Seq(
+      "com.github.os72" % "protoc-jar" % protobufVersion
```

Comment:
This jar pulls in the binary compiler which we don't need as part of the library.

You should be depending on `protobuf-java` instead as `Provided`?
https://github.com/nevillelyh/protobuf-generic/blob/master/build.sbt#L16

Reply:
Also see https://github.com/nevillelyh/protobuf-generic/blob/master/project/plugins.sbt#L20